zookeeper-server/src/main/java/org/apache/zookeeper/ZooKeeper.java

```diff
@@ -3398,6 +3403,26 @@ private ClientCnxnSocket getClientCnxnSocket() throws IOException {
         }
     }
 
+    /**
+     * Return the default watcher of this instance if requested.
+     *
+     * @param requested if the default watcher requested
+     * @return the default watcher if request, otherwise {@code null}.
+     * @throws IllegalStateException if a null default watcher is requested
+     */
+    @Nullable
+    private Watcher requestDefaultWatcher(boolean requested) {
```

Comment:
I would name this method
getDefaultWatcher
and the parameter 'required'

Reply:
Done.